lib/cog/bootstrap.ex

```diff
@@ -141,6 +158,19 @@ defmodule Cog.Bootstrap do
     :not_configured
   end
 
+  defp maybe_bootstrap_chat_handle_from_env(_user, nil),
+    do: Logger.info("No chat handle specified for bootstrap user; skipping")
+  defp maybe_bootstrap_chat_handle_from_env(user, handle) do
+    {:ok, provider_name} = Cog.Util.Misc.chat_adapter_module()
+    case Cog.Repository.ChatHandles.set_handle(user, provider_name, handle) do
+      {:ok, _} ->
+        Logger.info("Associated bootstrap user with chat handle '#{handle}'")
+        :ok
+      {:error, reason} ->
+        Logger.error("Could not associate bootstrap user with chat handle '#{handle}': #{reason}")
```

Comment:
I think we could stand to be more verbose here so the user knows exactly what to do.

```
Logger.error("""
Failed to create chat handle '#{handle}' for bootstrap user. Chat handle will need to be configured manually.
Error: #{reason}
""")
```

Or something like it.